ui/src/components/FacetCard.js

```diff
@@ -21,7 +21,7 @@ class FacetCard extends Component {
 
```

Comment:
Can isUnselected be shouldBeGray or isDimmed? I keep thinking it's "should checkbox be unselected".

Reply:
Done